status.go

```diff
@@ -46,6 +46,11 @@ var relatedObjects = []osconfigv1.ObjectReference{
 		Resource: "certificatesigningrequests",
 		Name:     "",
 	},
+	{
+		Group:    "rbac.authorization.k8s.io",
+		Resource: "clusterroles",
```

Comment:
Is a ClusterRole all you need?  I'd expect you'd also need a ClusterRoleBinding that attaches that role to a ServiceAccount?  And maybe other things?